engine/class_modules/sc_death_knight.cpp

```diff
@@ -1149,46 +1150,44 @@ inline void rune_event_t::execute_event()
-                           -> set_default_value( p -> conduits.debilitating_malady.percent() );
-
-  debuff.everfrost         = make_buff( *this, "everfrost", p -> find_spell( 337989 ) )
-                           -> set_default_value( p -> conduits.everfrost.percent() );
-
-  debuff.unholy_blight     = make_buff( *this, "unholy_blight_debuff", p -> talent.unholy_blight -> effectN( 1 ).trigger() )
-                           -> set_default_value_from_effect( 2 );
-
+inline death_knight_td_t::death_knight_td_t( player_t* target, death_knight_t* player )
+  : actor_target_data_t( target, player )
+{
+  dot.blood_plague         = target->get_dot( "blood_plague", player );
+  dot.frost_fever          = target->get_dot( "frost_fever", player );
+  dot.virulent_plague      = target->get_dot( "virulent_plague", player );
+  dot.soul_reaper          = target->get_dot( "soul_reaper", player );
+  dot.shackle_the_unworthy = target->get_dot( "shackle_the_unworthy", player );
+
+  debuff.mark_of_blood =
+      make_buff( *this, "mark_of_blood", player->talent.mark_of_blood )->set_cooldown( 0_ms );  // Handled by the action
+  debuff.razorice = make_buff( *this, "razorice", player->spell.razorice_debuff )
+                        ->set_default_value_from_effect( 1 )
+                        ->set_period( 0_ms );
+  debuff.festering_wound = make_buff( *this, "festering_wound", player->spell.festering_wound_debuff )
+                               ->set_cooldown( 0_ms )  // Handled by death_knight_t::trigger_festering_wound()
+                               ->set_stack_change_callback( [ player ]( buff_t*, int old_, int new_ ) {
+                                 // Update the FW target count if needed
+                                 if ( old_ == 0 )
+                                   player->festering_wounds_target_count++;
+                                 else if ( new_ == 0 )
+                                   player->festering_wounds_target_count--;
+                               } );
+
+  debuff.apocalypse_death = make_buff( *this, "death", player->find_spell( 327095 ) );  // Effect not implemented
+  debuff.apocalypse_famine =
+      make_buff( *this, "famine", player->find_spell( 327092 ) )->set_default_value_from_effect( 1 );
+  debuff.apocalypse_war = make_buff( *this, "war", player->find_spell( 327096 ) )->set_default_value_from_effect( 1 );
+
+  debuff.debilitating_malady = make_buff( *this, "debilitating_malady", player->find_spell( 338523 ) )
+                                   ->set_default_value( player->conduits.debilitating_malady.percent() );
+
+  debuff.everfrost = make_buff( *this, "everfrost", player->find_spell( 337989 ) )
+                         ->set_default_value( player->conduits.everfrost.percent() );
+
+  debuff.unholy_blight =
+      make_buff( *this, "unholy_blight_debuff", player->talent.unholy_blight->effectN( 1 ).trigger() )
+          ->set_default_value_from_effect( 2 );
```

Comment:
I fail to see how this is an improvement over the previous version. No alignement and inconsistent line breaks.

Reply:
The main idea was to have the function argument names consistent to its declaration. But I guess relying on auto-formatting after applying those changes was a bad idea to do in the DK module?

That said, having the second argument aligned for the get_dot function - and is definitely not consistent with how make_buff function calls are performed.

Things like mark of blood and apocalypse_famine and are a bit unfortunate, I agree.